src/modules/Dashboard/components/TakeNowModal/LabeledDropdown/LabeledUserDropdown.tsx

```diff
@@ -64,42 +71,105 @@ export const LabeledUserDropdown = ({
+
+  let groupBy: LabeledUserDropdownProps['groupBy'];
+
+  if (showGroups) {
+    groupBy = (option: ParticipantDropdownOption) => {
+      if (option.tag !== 'Team') {
+        return t('takeNow.modal.dropdown.participantGrouping');
+      } else {
+        return t('takeNow.modal.dropdown.teamMemberGrouping');
+      }
+    };
+  }
+
   return (
-    <StyledFlexColumn sx={{ gap: 1.6, marginTop: 2.4 }}>
+    <StyledFlexColumn sx={{ gap: 1.6, ...sx }}>
       <Box sx={{ display: 'flex', gap: 0.4 }}>
-        <StyledTitleMedium fontWeight="bold">{label}</StyledTitleMedium>
-        <Tooltip tooltipTitle={tooltip}>
-          <Box sx={{ height: 24 }}>
-            <StyledTitleTooltipIcon
-              sx={{ marginLeft: 0 }}
-              id="more-info-outlined"
-              width={24}
-              height={24}
-              data-testid={`${rest['data-testid']}-tooltip-icon`}
-            />
-          </Box>
-        </Tooltip>
+        <StyledTitleMedium sx={{ fontWeight: 700, color: variables.palette.on_surface }}>
+          {label}
+        </StyledTitleMedium>
+        <RenderIf condition={!!tooltip}>
+          <Tooltip tooltipTitle={tooltip}>
+            <Box sx={{ height: 24 }}>
+              <StyledTitleTooltipIcon
+                sx={{ marginLeft: 0 }}
+                id="more-info-outlined"
+                width={24}
+                height={24}
+                data-testid={`${rest['data-testid']}-tooltip-icon`}
+              />
+            </Box>
+          </Tooltip>
+        </RenderIf>
       </Box>
       <Autocomplete
-        renderInput={(params) => {
+        renderInput={(params: AutocompleteRenderInputParams) => {
           const { InputLabelProps: _InputLabelProps, ...rest } = params;
 
           return <TextField {...rest} placeholder={placeholder} name={name} />;
         }}
+        sx={{
+          '& .MuiInputBase-root': {
+            borderBottomLeftRadius: shouldShowWarningMessage ? 0 : variables.borderRadius.sm,
+            borderBottomRightRadius: shouldShowWarningMessage ? 0 : variables.borderRadius.sm,
+          },
+        }}
         options={combinedOptions}
-        renderOption={({ children: _children, ...props }, { id, ...psProps }) => (
+        renderOption={(
+          { children: _children, ...props },
+          { id, tag, secretId, nickname, ...psProps },
+        ) => (
           <ParticipantSnippet<'li'>
             key={id}
+            tag={tag}
+            secretId={tag === 'Team' ? nickname : secretId}
+            nickname={tag === 'Team' ? null : nickname}
             {...psProps}
             boxProps={{
-              sx: { p: theme.spacing(0.6, 1.6), cursor: 'pointer' },
+              sx: {
+                p: theme.spacing(0.6, 1.6),
+                cursor: 'pointer',
+                ':last-child': {
+                  borderBottom: `${variables.borderWidth.md} solid ${variables.palette.surface_variant}`,
+                },
```

Comment:
This is a total nit, but I noticed that the last group has a bottom border, and I don't think it should:

<img width="541" alt="Screenshot 2024-05-22 at 3 03 42 PM" src="https://github.com/ChildMindInstitute/mindlogger-admin/assets/1282772/b8471a79-cd6b-44d0-a0be-ea1b95021c94">


So instead of applying it here in the `renderOption` callback, you could apply it in `renderGroup` below:

```tsx
        renderGroup={(params) => {
          const { key, group, children } = params;

          return (
            <Box
              component="li"
              key={key}
              sx={{
                '&:not(:last-child)': {
                  borderBottom: `${variables.borderWidth.md} solid ${variables.palette.surface_variant}`,
                },
              }}
            >
              <StyledLabelBoldLarge
                sx={{
                …
```

Reply:
Very sharp on this one. Thanks man